src/Adapters/Logger/WinstonLoggerAdapter.js

```diff
@@ -60,6 +60,22 @@ export class WinstonLoggerAdapter extends LoggerAdapter {
     return logger.error.apply(undefined, arguments);
   }
 
+  warn() {
+    return logger.warn.apply(undefined, arguments);
+  }
+
+  verbose() {
+    return logger.verbose.apply(undefined, arguments);
+  }
+
+  log() {
```

Comment:
agreed. 

In winston, the 1st argument of log is the level you want to log to. 

As for silly, I can down to verbose as we don't log higher